src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -70,7 +70,9 @@ internal float WorldPositionX
             get
             {
                 float temp = 0.0f;
-                GetProperty(View.Property.WORLD_POSITION_X).Get(out temp);
+                PropertyValue _wordPositionX = GetProperty(View.Property.WORLD_POSITION_X);
+                _wordPositionX?.Get(out temp);
```

Comment:
could you check this NUI ground rule suggestion #2 ? (https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule)
How do you think about using camel case for private field's naming.

Reply:
Done